docker-compose.yaml

```
@@ -0,0 +1,113 @@
+      retries: 10
+    secrets:
+      - mysql_root_password
+      - mysql_password
+
+  server:
+    build: .
+    container_name: switchmap-server
+    environment:
+      - CONTAINER_ROLE=server
+    ports:
+      - "7000:7000"  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)